tool/check-code.sh

```diff
@@ -42,9 +42,14 @@ flutter --version
 if [[ $REFRESH ]]; then
   echo "=> Refreshing code excerpts..."
   (
+    commitHash=$(git reflog -n1 --format='%h')
```

Comment:
Could you put this new behavior under a new flag, like `--refresh-diff` and then specify that in `test.sh`? So that those running this locally can opt in to the behavior.

Reply:
Should we opt-in? IMO it should at least opt-out. This is helpful whether we test locally or remotely.

Reply:
That's fine too. It would just be nice to have some way to disable it.